src/ui/types.ts

```diff
@@ -21,6 +23,7 @@ export interface Reply {
 	isMyReply?: boolean
 	tokenName?: string
 	user: User
+	keyExchange: KeyExchange
```

Comment:
Just a mental note for self: we should have a UI version of this without the imported `KeyExchange` type and a non-UI version that extends it. The reason being is that the idea behind `/ui` folder is to extract it all to the UI library at some point in future. Maybe.